styles.css

```diff
@@ -0,0 +1,78 @@
+html,
+body {
+  height: 100%;
+  font-family: 'Open Sans', sans-serif;
+  text-align: center;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  height: 475px;
+  width: 725px;
+  box-sizing: border-box;
+}
+
+.profile {
+  display: inline-block;
+  height: 415px;
+  width: 313px;
+  margin: 15px;
+  border: 2px solid #c9c9c9;
+  border-top: 0px;
+  border-radius: 10px;
+  padding-bottom: 10px;
+}
+
+.pic {
+  width: 315px;
+  height: 250px;
+  border: 0px;
+  margin: 0 -2px 0 -3px;
+  border-radius: 8px 8px 0px 0px;
+  position: relative;
+  overflow: hidden;
+}
+
+.name {
+  font-size: 30px;
+  padding-top: 10px;
+  }
+
+.title {
+  padding: 10px;
+}
+.separator {
+  border: 1px solid #c9c9c9;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.funds {
+  display: inline-block;
+  height: auto;
+  width: auto;
+  position: relative;
+  top: 5px;
+  right: 45px;
+}
+
+.donors {
+  display: inline-block;
+  height: auto;
+  width: auto;
+  position: relative;
+  top: 5px;
+  left: 45px;
```

Comment:
while this may work here and on 58-59, I betcha there's a simpler way to accomplish this goal